nifi-assembly/LICENSE

```diff
@@ -1085,29 +1085,3 @@ information can be found here: http://www.adobe.com/devnet/xmp/library/eula-xmp-
    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
    OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
    THE POSSIBILITY OF SUCH DAMAGE.
-
```

Comment:
@mattyb149 why are we removing the JSoup clause from the LICENSE? I believe that is required by the nifi-html-processors module